src/main/paradox/docs/index.md

```diff
@@ -53,20 +53,20 @@ This service is the heart of the BlueBrain Nexus platform. It allows users to de
 knowledge graph with data, attach files to data. It also provides semantic search facilities to discover similar and
 relevant data in the platform.
 
-[Source Code](https://github.com/BlueBrain/nexus-kg) | [Documentation](./kg)
+[Source Code](https://github.com/BlueBrain/nexus-kg) | @ref:[Documentation](./api/kg/index.md)
```

Comment:
why the `@ref`?

Reply:
[@ref](https://developer.lightbend.com/docs/paradox/current/directives/linking.html#ref-link) is going to verify that the link is correct when running `paradox` command, avoiding wrong links

Reply:
then please also add it to admin link